### PR TITLE
the spell check

### PR DESCRIPTION
**Pull request by shiyuan1992**
the   spell  check

### CODE REVIEW

**Comment by shiyuan1992** [comment]
Have signed CLA

File: PROCESSES.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ description: DEPRECATED foo bar baz qux...
...
+When new maintainers are interested in bring a chart out of depreciation with
 new features or new support that can be an option. To un-deprecate a chart:
 
 1. Update the maintainers on the chart if any are listed. The previous maintainers should not be expected to maintain the chart unless they explicitly decide to do so.
 1. If there is an OWNERS file in the chart that should be updated with the new reviewers and  approvers.
 1. The deprecated property from the `Chart.yaml` file should be removed along with any associated comment.
-1. The chart `version` needs to be incremented accordingly. If the same functionality is kept the version can be a patch increase. Otherwise the minor or major version needs to be incremented. For more detail on changing the version number see the [semver specification](http://semver.org).
+1. The chart `version` needs to be incremented accordingly. If the same functionality is kept the version can be a patch increase. Otherwise the minor or major version needs to be incremented. For more detail on changing the version number see the server specification](http://semver.org).
```
Comment (mattfarina) [inline PROCESSES.md:36]: The `[` in front of semver was to make it part of the link. Not it's a broken link. Can you undo this change?

**Review by mattfarina — CHANGES_REQUESTED** [review]
Thanks for fixing the spelling mistakes.

The `version` in the Chart.yaml file will need to be incremented as well.

File: PROCESSES.md (hunk 1 above)
Comment (shiyuan1992) [reply PROCESSES.md:36]: OK, I will change it.

**Comment by cpanato** [comment]
looks like the repo fork repo does not exist.

/close

**Comment by mattfarina** [comment]
Closing due to being stale
